scipy/stats/_continuous_distns.py

```diff
@@ -6021,12 +6024,10 @@ def _argcheck(self, skew):
         return np.ones(np.shape(skew), dtype=bool)
 
     def _stats(self, skew):
-        _, _, _, _, _, beta, alpha, zeta = (
-            self._preprocess([1], skew))
-        m = zeta + alpha / beta
-        v = alpha / (beta**2)
-        s = 2.0 / (alpha**0.5) * np.sign(beta)
-        k = 6.0 / alpha
+        m = 0
+        v = 1
```

Comment:
Maybe these are supposed to be vectorized? It's hard to tell what this should be when none of the distributions are accepting array inputs as they're supposed to (gh-12192).